app/services/data/get-current-workload-report.js

```diff
@@ -0,0 +1,11 @@
+const config = require('../../../knexfile').app
+const knex = require('knex')(config)
+
+module.exports = function () {
+  return knex('workload_report')
+    .whereNull('effective_to')
```

Comment:
Also include where effective_from is not null? For consistency / extra check that it's a valid record

Reply:
added